src/display/editor/stamp.js

```diff
@@ -141,6 +157,12 @@ class StampEditor extends AnnotationEditor {
       this._uiManager.useNewAltTextFlow &&
       this.#bitmap
     ) {
+      this._reportTelemetry({
+        action: "pdfjs.image.image_added",
+        data: {
+          alt_text_modal: false,
+        },
```

Comment:
Nit: This fits on one line.
```suggestion
        data: { alt_text_modal: false },
```

Reply:
Could we add a lint rule to enforce that ?

Reply:
Unfortunately not, as far as I can tell, since this formatting is controlled by Prettier.